### PR TITLE
refactor: remove dispatch_json.js from cli/rt and cli/tsc

### PR DESCRIPTION
**Pull request by bartlomieju**
Instead use Deno.core.jsonOpSync and Deno.core.jsonOpAsync

### CODE REVIEW

File: core/core.js [hunk 1]
```diff
@@ -208,14 +208,14 @@ SharedQueue Binary Layout
   }
 
   function decodeJson(ui8) {
-    const s = Deno.core.decode(ui8);
```
Comment (bartlomieju) [inline core/core.js:211]: This binding made json dispatch defined in this file work only when `Deno` global was available - so not working in workers by default

File: cli/tests/unit/buffer_test.ts [hunk 2]
```diff
@@ -329,9 +329,9 @@ unitTest(async function bufferReadFromSync(): Promise<void> {
 
 unitTest(async function bufferTestGrow(): Promise<void> {
   const tmp = new Uint8Array(72);
-  for (const startLen of [0, 100, 1000, 10000, 100000]) {
+  for (const startLen of [0, 100, 1000, 10000]) {
     const xBytes = repeat("x", startLen);
-    for (const growLen of [0, 100, 1000, 10000, 100000]) {
+    for (const growLen of [0, 100, 1000, 10000]) {
```
Comment (ry) [inline cli/tests/unit/buffer_test.ts:334]: Unrelated?

Comment (bartlomieju) [reply cli/tests/unit/buffer_test.ts:334]: Yeah, but I hate waiting 10s on a single unit test... I can revert

**Review by ry — APPROVED** [review]
LGTM
